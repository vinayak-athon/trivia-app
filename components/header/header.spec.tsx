import React from "react";
import { render, cleanup } from "@Test/utils";
import { Header } from "@Components";

afterAll(cleanup);

describe("Components", () => {
  describe("Header", () => {
    const { container } = render(<Header />);
    it("should match snapshot", () => {
      expect(container).toMatchSnapshot();
    });
  });
});
