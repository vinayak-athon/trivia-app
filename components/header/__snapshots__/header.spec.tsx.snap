// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Header should match snapshot 1`] = `
.c2 {
  cursor: pointer;
  color: #000000;
  background: #f9da2e;
  border-radius: 5px;
  padding: 5px 10px;
  font-family: Arial;
  font-weight: 700;
  font-size: 25px;
  -webkit-letter-spacing: 0;
  -moz-letter-spacing: 0;
  -ms-letter-spacing: 0;
  letter-spacing: 0;
  text-transform: uppercase;
}

.c3 {
  background-color: green;
  color: inherit;
  font-size: 10px;
  border: none;
}

.c4 {
  background-color: inherit;
  color: inherit;
  font-size: 10px;
  border: none;
}

.c1 {
  cursor: pointer;
}

.c0 {
  color: #ffffff;
  background-color: #000000;
  text-align: center;
  text-transform: uppercase;
  font-weight: bold;
  padding: 0.625rem 0;
  width: 100%;
}

<div>
  <header
    class="c0"
  >
    <h1
      class="c1"
    >
      common:AppName
    </h1>
    <button
      class="c2 c3"
    >
      English
    </button>
    <button
      class="c2 c4"
    >
      Spanish
    </button>
  </header>
</div>
`;
